shell/browser/net/electron_url_loader_factory.cc

```diff
@@ -294,14 +296,13 @@ void ElectronURLLoaderFactory::StartLoading(
     // module.
     //
     // I'm not sure whether this is an intended behavior in Chromium.
-    mojo::Remote<network::mojom::URLLoaderFactory> proxy_factory_remote(
-        std::move(proxy_factory));
+    if (proxy_factory.is_valid()) {
+      mojo::Remote<network::mojom::URLLoaderFactory> proxy_factory_remote(
+          std::move(proxy_factory));
```

Comment:
You shouldn't bind invalid `mojo::PendingRemote` objects. For some reason the `mojo` code doesn't do that `DCHECK` if you bind via the constructor, rather than doing `proxy_factory_remote.Bind(std::move(proxy_factory))`. Binding then checking if bound is not a proper code path, the correct way is to check for validity first then bind.